report/src/discussion/discussion.tex

```diff
@@ -0,0 +1,38 @@
+\chapter{Discussion}\label{chap:discussion}
+The project involved two main phases: initially, we designed LoFi and HiFi prototypes for a tablet-based meeting room management system, which were well-received by the stakeholder, Mariendal IT.
+However, through the usability test, it became evident that automating presence detection was crucial to fully address their needs.
+This led to the problem formulation in section \ref{sec:problem_statement}.
+
+In chapter \ref{chap:presence_intro}, we presented five metrics that would be used to evaluate a selection of technologies for presence detection.
+Based on our research, we determined that BLE satisfied most of these metrics in relation to our stakeholders problem.
+Using BLE allowed us to consider several techniques for presence detection purposes of which we considered lateration and fingerprinting as primary candidates.
+Due to the need for non-noisy environments for lateration to work, we determined that the most appropiate technique would be fingerprinting along with kNN for classification.
+Based on this, we implemented a solution utilizing BLE beacons and fingerprinting and in chapter \ref{chap:system_design}, we presented our results and our evaluation of these results. 
+We found that utilizing four beacons to outline a room and a map generated by traversing the room in a grid pattern of $1m \times 1m$ we were able to achieve significant results.
+Depending on the chosen \textit{k} for nearest neighbor search, we were able to achieve between $92.86$ and $95.71$ percent accuracy for the inside case and between $48$ and $88$ percent accuracy for the outside scenario. 
+As mentioned in \ref{sec:ChoiceOfTechnique}, the goal of the project was not to achieve high accuracy and precision on the exact positioning of people in a room. 
+The goal was rather to investigate whether we could detect the presence of people in a meeting with reasonable certainty.
+
+These findings align with what similar research has concluded; that using kNN is sufficiently accurate, and that complex machine learning is not necessary to obtain useful results.\cite{ble_kneares_neural}
+
+The experimental setup depicted in Figure~\ref{fig:experiment_setup} places four beacons within a room.
+The resultant map was generated by a systematic grid-based traversal of this space.
```

Comment:
Just a few paragraphs above we described this grid-based approach. Should we refer back or is it fine to mention a second time like this for precisions sake?